src/main/scala/mesosphere/marathon/core/externalvolume/providers/DVDIProvider.scala

```diff
@@ -18,19 +18,19 @@ import scala.collection.JavaConverters._
   *   - docker containerizer requires that referenced volumes be created prior to application launch
   *   - mesos containerizer only supports volumes mounted in RW mode
   */
-protected[externalvolume] case object DVDIProvider
-    extends AbstractExternalVolumeProvider("dvdi") with DVDIProviderValidations {
+protected[externalvolume] case object DVDIProvider extends ExternalVolumeProvider {
+  override val name: String = "dvdi"
```

Comment:
In this case it does not matter. A val takes up memory for every instance, a def "calculates" the value every time. If it is a static string and there is only one instance, it does not matter.